smolyay/smolyak.py

```diff
@@ -0,0 +1,141 @@
+import numpy as np
```

Comment:
In published code, you should try not to alias packages unless you really need to because it pollutes the namespace:
```suggestion
import numpy
```
In your own scripts, it's perfectly fine to do this though

Reply:
Done.